synfig-studio/src/gui/docks/dock_toolbox.cpp

```diff
@@ -232,6 +235,25 @@ Dock_Toolbox::add_state(const Smach::state_base *state)
 	refresh();
 }
 
+void 
+Dock_Toolbox::button_new_layer_creator()
+{
+	Gtk::RadioToolButton *tool_button = manage(new Gtk::RadioToolButton());
+	tool_button->set_group(radio_tool_button_group);
```

Comment:
Does it need to be a `RadioToolButton` instead of a regular one?

Reply:
- sorry was having exams
- i made it like the already exist button in layer panel 
- because too , to make all things unified as all toolbox buttons and their radio button group 
- but if u have different opinion point to me where to search  as from my search it can be many things